site/docs/patterns/international-address-input.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: International Address Input
+layout: DetailPattern
+sidebar:
+  label: International Address Input
+aliases:
+  - /salt/patterns/international-address-input
+---
+
+International Address Input is a pattern that gives users a set of form fields to input address information for specific countries.
+
+## Pattern scope
+
+The pattern is a form that displays relevant fields for a user-selected location. You can use this pattern to provide a consistent look and feel when implementing International Address Input interactions across your applications. Future versions of the pattern will cover use of APIs, use of autocompletion, and enhanced support for more locations.
+
+The pattern aims to accommodate users worldwide, but your team may be able to take further steps that fall outside the scope of this pattern. For example, check with your tech partners if it is possible to detect user location and automatically adjust the form.
+
+## When to use it
+
+Use the pattern when your user needs to enter an address for a specific country. This could be for shipping, contact, legal or compliance purposes, such as for a client onboarding flow or client detail collection process.
+
+## Pattern content
+
+The pattern consists of Salt Form Field components (for layout guidance, please refer to the [Forms pattern](./forms)). It comprises two formats: Dynamic and Generic.
```

Comment:
"Form Field" is a specific component, needs update. 
Something like "various form controls"

Reply:
asked in the Figma